docs/source/examples/Widget List.ipynb

```diff
@@ -929,6 +929,14 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `icon` attribute can be used to define an icon; see the [fontawesome](https://fontawesome.com/v4.7.0/icons/) page for available icons. \n",
```

Comment:
On current master, we have upgraded things to use fontawesome 5. Can you link to `https://fontawesome.com/icons` instead?